fdm-app/app/components/custom/atlas-fields-sources.tsx

```
@@ -54,93 +61,103 @@ export function AvailableFieldsSource({ url, zoomLevelFields, children }: { url:
+    setSelectedFieldsData,
+    children,
+}: {
+    selectedFieldsData: FeatureCollection
+    setSelectedFieldsData: React.Dispatch<
+        React.SetStateAction<FeatureCollection>
+    >
+    children: JSX.Element
+}) {
+    const { current: map } = useMap()
 
     useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: map

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.

Either include them or remove the dependency array

(lint/correctness/useExhaustiveDependencies)